assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -271,6 +271,9 @@ storage:
           -A INPUT -p tcp --dport 2380 -j ACCEPT
           -A INPUT -p tcp --dport 2381 -j ACCEPT
           -A INPUT -p tcp --dport 6443 -j ACCEPT
+          %{~ if controllers == "1" ~}
+          -A INPUT -p tcp --dport 7443 -j ACCEPT
```

Comment:
This seems tricky as calico is not overriding (yet?) these rules, but this rule is not on calico global network policies. If we need the rule during the cluster lifecycle (as I understand from https://github.com/kinvolk/lokomotive/pull/32/files#r390412048), I think we should re-add it there. But, if it's only for internal traffic, add it only for internal traffic.

Not sure, though, if it makes sense to duplicate the rule in calico and here... Maybe we want in case there are issues with calico?

Reply:
Keep in mind, that kube-apiserver will only listen on host internal IP, not on the external one, so this rule is no-op for external IP address. I'd say it's fine'ish for now

Reply:
Should we create a tech-debt issue to follow up on this later?

Reply:
Yes. I'll create it.

Reply:
Thanks, please link and mark as resolved :)

Reply:
Created #137.